articles/operator-nexus/reference-near-edge-storage.md

```diff
@@ -21,7 +21,11 @@ This table lists the available SKUs for the storage appliance in the near-edge A
 
 | SKU                     | Description                            |
 | ----------------------- | ------------------------------------- |
-| Pure x70r3-91           | Storage appliance model x70r3-91 provided by Pure Storage |
+| Pure x20r4-22           | Storage appliance model x20r4-22 provided by Pure Storage |
```

Comment:
@mbustame can we also update the section "Storage Limits" here at the bottom pls. Also, does it have any impact on the connectivity as reflected in the diagram in next section.